docs/content/docs/testing/errors.md

```diff
@@ -0,0 +1,290 @@
+---
+title: Common Fluid Framework Errors
+menuPosition: 5
+---
+
+This article lists the most common errors Fluid developers will encounter. The first steps to resolve the errors are listed. If you get an error that is not included here, please [let us know with a GitHub issue](https://github.com/microsoft/FluidFramework/issues) and we’ll update both the documentation and our guidance. 
+
+### **`<X>` is not a constructor** 
```

Comment:
I don't know what flavor of Markdown is used by the Hugo publishing system, but usually the "###" symbols for headers control whether the line is bolded, so the "**" here are probably unneeded.

Reply:
Good point Ricky! I tried using ## but the error headings were not being readable. It just felt like a long doc with unbolded headings that made it difficult to navigate from one error to another. Let me check again if I can see how the headings readability can be improved without using both # and *